package.json

```diff
@@ -235,6 +235,7 @@
     "cross-fetch": "^2.2.6",
     "trim-newlines": "^3.0.1",
     "aws-sdk": "^2.1141.0",
+    "aws-cdk-lib": "2.129.0",
```

Comment:
What is the purpose of this declaration, and why is it pinned rather than compatible?